supertokens_python/recipe/thirdparty/providers/gitlab.py

```diff
@@ -15,89 +15,15 @@
 from __future__ import annotations
 
 from supertokens_python.recipe.thirdparty.provider import Provider
-from supertokens_python.recipe.thirdparty.types import (
-    AccessTokenAPI,
-    AuthorisationRedirectAPI,
-    UserInfo,
-    UserInfoEmail,
-)
+from .custom import GenericProvider, NewProvider
+from ..provider import Provider, ProviderInput
 
-if TYPE_CHECKING:
-    from supertokens_python.framework.request import BaseRequest
 
+# TODO Implement when it's done in Node PR
```

Comment:
remove all todos from code and transfer them to https://github.com/supertokens/supertokens-python/pull/276

Reply:
Done.